spec/defines/filter/client_auth_n_spec.rb

```diff
@@ -190,8 +190,9 @@
           'pass' => 'password',
           'uri'  => 'http://uri'
         },
-        :client_maps => [ 'testing', ],
-        :delegating  => 'true'
+        :client_maps         => [ 'testing', ],
+        :delegating          => 'true',
+        :delegating_quality  => '0.9'
```

Comment:
OK, i was confused for a moment.  What's the issue again with the delgating_quality?  I'm setting it in order to verify its output later in the test.